code/game/gamemodes/changeling/implements/powers/stings.dm

```diff
@@ -17,30 +24,72 @@
 	if(!changeling_power(required_chems))
 		return
 	if(T.isSynthetic())
-		to_chat(src, "<span class='warning'>[T] is not compatible with our biology.</span>")
+		to_chat(src, SPAN_WARNING("[T] is not compatible with our biology."))
 		return
 
 	changeling.chem_charges -= required_chems
 	changeling.sting_range = 1
-	src.verbs -= verb_path
+	verbs -= verb_path
 	ADD_VERB_IN(src, 10, verb_path)
 
 	if(stealthy)
-		to_chat(src, "<span class='notice'>We stealthily sting [T].</span>")
-		to_chat(T, "<span class='warning'>You feel a tiny prick.</span>")
+		to_chat(src, SPAN_NOTICE("We stealthily sting [T]."))
+		to_chat(T, SPAN_WARNING("You feel a tiny prick."))
 	else
-		src.visible_message(pick("<span class='danger'>[src]'s eyes balloon and burst out in a welter of blood, burrowing into [T]!</span>",
-								"<span class='danger'>[src]'s arm rapidly shifts into a giant scorpion-stinger and stabs into [T]!</span>",
-								"<span class='danger'>[src]'s throat lengthens and twists before vomiting a chunky red spew all over [T]!</span>",
-								"<span class='danger'>[src]'s tongue stretches an impossible length and stabs into [T]!</span>",
-								"<span class='danger'>[src] sneezes a cloud of shrieking spiders at [T]!</span>",
-								"<span class='danger'>[src] erupts a grotesque tail and impales [T]!</span>",
-								"<span class='danger'>[src]'s chin skin bulges and tears, launching a bone-dart at [T]!</span>"))
+		src.visible_message(pick(SPAN_DANGER("[src]'s eyes balloon and burst out in a welter of blood, burrowing into [T]!"),
+								SPAN_DANGER("[src]'s arm rapidly shifts into a giant scorpion-stinger and stabs into [T]!"),
+								SPAN_DANGER("[src]'s throat lengthens and twists before vomiting a chunky red spew all over [T]!"),
+								SPAN_DANGER("[src]'s tongue stretches an impossible length and stabs into [T]!"),
+								SPAN_DANGER("[src] erupts a grotesque tail and impales [T]!"),
+								SPAN_DANGER("[src]'s chin skin bulges and tears, launching a bone-dart at [T]!")))
 
 	if(!T.mind || !T.mind.changeling)
 		return T	//T will be affected by the sting
 	return
 
+/mob/proc/apply_sting_effects(var/mob/living/carbon/target, var/type, var/strength)
```

Comment:
A bit confused over this. Why did you make the sting effect a mob instead of being part of the sting datum?